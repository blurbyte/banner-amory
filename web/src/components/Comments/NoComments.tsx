import styled from 'styled-components';

const NoComments = styled.p`
  margin-top: 2rem;
  font-style: italic;
`;

export default NoComments;
