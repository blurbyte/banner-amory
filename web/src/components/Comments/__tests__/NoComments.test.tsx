import * as React from 'react';
import * as renderer from 'react-test-renderer';
import 'jest-styled-components';

import NoComments from '../NoComments';

test('renders correctly', () => {
  const component = renderer.create(<NoComments>There are no comments here!</NoComments>);
  expect(component.toJSON()).toMatchSnapshot();
});
