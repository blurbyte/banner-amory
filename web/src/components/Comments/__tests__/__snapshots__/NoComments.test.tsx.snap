// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  margin-top: 2rem;
  font-style: italic;
}

<p
  className="c0"
>
  There are no comments here!
</p>
`;
